db/hints/manager.cc

```diff
@@ -174,6 +174,9 @@ void manager::register_metrics(const sstring& group_name) {
         sm::make_counter("sent", _stats.sent,
                         sm::description("Number of sent hints.")),
 
+        sm::make_gauge("sent_total_size", _stats.sent_hints_size,
```

Comment:
Assuming that it's the total bytes ever sent, it should be counter (ever growing) and a better name would be sent_bytes_total